src/main/scala/devices/debug/SBA.scala

```diff
@@ -100,13 +101,13 @@ object SystemBusAccessModule
     val hasSBAddr3 = (sb2tl.module.edge.bundle.addressBits >= 97)
     val hasAddr    = Seq(true, hasSBAddr1, hasSBAddr2, hasSBAddr3)
 
-    val SBADDRESSFieldsReg = Seq.fill(4)(Reg (UInt(32.W)))
+    val SBADDRESSFieldsReg = RegInit(VecInit(Seq.fill(4) {0.U(32.W)} ))
```

Comment:
this is another place where it seems like we are adding functionality, not just changing to chisel3. Is the change to RegInit intentional?

Reply:
Yes, FIRRTL complained about these being not fully initialized.  The code below this only assigns the portion of the declared Reg that is actually used in this config, leaving the rest of it unassigned, resulting in that error.

Reply:
So are we Ok with this being reset by the module reset? This is a change in functionality.

Reply:
You're right, this did change functionality.  I changed to a different method that should maintain existing function.